homeassistant/components/nmbs/sensor.py

```diff
@@ -92,7 +92,7 @@ def __init__(self, api_client, live_station):
         """Initialize the sensor for getting liveboard data."""
         self._station = live_station
         self._api_client = api_client
-
+        self._unique_id = f"nmbs_live_{self._station}"
```

Comment:
You don't need to add `nmbs_live`. All unique IDs are scoped to integration/platform. So `nmbs/sensor/<unique_id>`

Reply:
Okay thanks for the tip! 👍

Reply:
There are more than one sensor type for this platform. Currently we only add unique_id for the live sensor, but it's good to not have to change the unique_id if we add it for the other sensor types too.